pkg/lib/cockpit-components-dropdown.tsx

```diff
@@ -21,7 +21,7 @@ import React, { useState } from 'react';
 import PropTypes from "prop-types";
 
 import { MenuToggle } from "@patternfly/react-core/dist/esm/components/MenuToggle";
-import { Dropdown, DropdownList } from "@patternfly/react-core/dist/esm/components/Dropdown";
+import { Dropdown, DropdownList, DropdownPopperProps } from "@patternfly/react-core/dist/esm/components/Dropdown";
```

Comment:
Wondering if we want to use `import type` not sure if get is useful because the current form wont get shook? Or does it?

Reply:
I think `import type` only makes sense on files that you wouldn't already import for non-type reasons.

ie: If you already have an `import` then don't add an `import type`.  But if you wouldn't otherwise have an `import`, and only want it for typing reasons, then `import type` is the one you want.